src/lib/bootloader/proposal_client.rb

```diff
@@ -46,12 +47,18 @@ def make_proposal(attrs)
 
       if Yast::Mode.update
         return { "raw_proposal" => [_("do not change")] } unless propose_for_update(force_reset)
+      elsif Yast::Bootloader.proposed_cfg_changed
+        # do nothing as user already modify it
       else
         # in installation always propose missing stuff
```

Comment:
OK, could you add this info as comments into the code? It would help reading the code...